src/tests/pokete/util/__init__.py

```
@@ -1 +1 @@
-from .semantic_version import *
+from .test_semantic_version import *
```

Ruff (0.14.14)

[error] `from .test_semantic_version import *` used; unable to detect undefined names

(F403)